freqtrade/freqtradebot.py

```diff
@@ -423,11 +423,14 @@ def _check_depth_of_market_buy(self, pair: str, conf: Dict) -> bool:
         order_book_bids = order_book_data_frame['b_size'].sum()
         order_book_asks = order_book_data_frame['a_size'].sum()
         bids_ask_delta = order_book_bids / order_book_asks
-        logger.info('bids: %s, asks: %s, delta: %s', order_book_bids,
-                    order_book_asks, bids_ask_delta)
+        logger.info('bids: %s, asks: %s, delta: %s, askprice: %s, bidprice: %s, immediate askquantity: %s, immediate bidquantity: %s',
+                    order_book_bids, order_book_asks, bids_ask_delta, order_book['asks'][0][0], order_book['bids'][0][0], order_book['asks'][0][1], order_book['bids'][0][1])
         if bids_ask_delta >= conf_bids_to_ask_delta:
+            logger.info('bids to ask delta DOES satisfy condition.')
             return True
-        return False
+        else:
+            logger.info('bids to ask delta DOES NOT satisfy condition.')
```

Comment:
I would avoid "screaming" in log messages (all uppercase words).  

Also, including the pair to this log message will make searching through logs easier as you can search for the pair that did / did not not buy ...

Reply:
Done, not sure why flake8 ended up modifying lots of other lines on the file though for length of line.